middleware/authorization.js

```diff
@@ -56,32 +57,40 @@ function generateRefreshToken(rowInfo) {
 async function reIssueToken(req, res, next) {
     const accessToken = verifyToken(req.headers["authorization"], ACCESS_PRIVATE_KEY);
     const refreshToken = verifyToken(req.headers["refresh"], REFRESH_PRIVATE_KEY);
-
-    if (accessToken === "TokenExpiredError" && refreshToken === "TokenExpiredError") {
+    console.log("accessToken", accessToken);
+    console.log("refreshToken", refreshToken);
+    if ((accessToken === "TokenExpiredError") && (refreshToken === "TokenExpiredError")) {
```

Comment:
```suggestion
    if ( !req.headers["authorization"] && !req.headers["refresh"] ){
      return res.status(403).json({message:"not authenticated"});
    }
    if (refreshToken === "TokenExpiredError") {
```

첫번째 조건의 추가로 두개의 헤더가 필수임을 명시 합니다.

두번째 조건으로 리프레시토큰이 만료됐는지 확인합니다.

accessToken은 refreshToken으로 생성할 수 있기때문에 refreshToken만 만료 확인을 하면됩니다.